packages/bot-web-ui/src/components/dashboard/dashboard-component/cards.tsx

```diff
@@ -1,62 +1,171 @@
-            method: () => setActiveTab(2),
-        },
-    ];
+const Cards = ({
+    handleFileChange,
+    setActiveTab,
+    onConfirmSave,
+    setLoadedLocalFile,
+    onDriveConnect,
+    loadFileFromLocal,
+    setFileLoaded,
+    has_file_loaded,
+    is_dialog_open,
+    showVideoDialog,
+    closeResetDialog,
+    dialog_options,
+}: TCardsProps) => {
+    type TCardArray = {
+        icon: string;
+        content: string;
+        method: () => void;
+        disable: string | '';
+    };
+    const openGoogleDriveDialog = () => {
+        showVideoDialog({
+            type: 'google',
+            component: <GoogleDrive />,
+            url: '',
+        });
+    };
+    const file_input_ref = React.useRef<HTMLInputElement | null>(null);
+    const [is_file_supported, setIsFileSupported] = React.useState<boolean>(true);
+
+    const clear_preview_ref = React.useRef<HTMLInputElement | null>(null);
+    const openFileLoader = () => {
+        file_input_ref?.current?.click();
+    };
+    const [icon_array, SetIconArray] = React.useState<TCardArray[] | []>([]);
+    React.useEffect(() => {
+        const IconArray: TCardArray[] = [
+            {
+                icon: 'IcMyComputer',
+                content: 'My computer',
+                method: openFileLoader,
+                disable: has_file_loaded ? 'disabled--card' : '',
+            },
+            {
+                icon: 'IcGoogleDriveDbot',
+                content: 'Google Drive',
+                method: openGoogleDriveDialog,
+                disable: has_file_loaded ? 'disabled--card' : '',
+            },
+            {
+                icon: 'IcBotBuilder',
+                content: 'Bot Builder',
+                method: () => setActiveTab(1),
+                disable: '',
+            },
+            {
+                icon: 'IcQuickStrategy',
+                content: 'Quick Strategy',
+                method: () => setActiveTab(2),
+                disable: '',
+            },
+        ];
+        SetIconArray(IconArray);
+    }, [has_file_loaded]);
 
     return (
-        <div className='dc-tabs__dashboard-cards-wrapper' id='db-dashboard-cards'>
-            {actions.map(({ icon, content, method }, index) => (
-                <div key={index} className='dc-tabs__dashboard-card'>
-                    <Icon
-                        className='dc-tabs__dashboard-card__image'
-                        width='8rem'
-                        height='8rem'
-                        icon={icon}
-                        id={icon}
-                        onClick={method}
-                    />
-                    <Text size='xxs' line_height='s' color='prominent'>
-                        {localize(content)}
-                    </Text>
-                </div>
-            ))}
+        <div className='tab__dashboard__table'>
+            <div
+                className='dc-tabs__content_group_tiles tab__dashboard__table__tiles'
```

Comment:
dc-tabs__content_group_tiles should be => dc-tabs__content-group__tiles
_group=> -group
_tiles=> __tiles

Reply:
done ✅